README.md

```diff
@@ -43,7 +43,12 @@ Finally `npm start`
 👤 **Kyrillos Hany**
 
 - GitHub: [@Bondok6](https://github.com/Bondok6)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kyrillos-hany/)
+
+👤 **Carlos Herver**
+
+- GitHub: [@Carlos](https://github.com/carloshs1994)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/carloshs94/)
```

Comment:
- [ ] **[OPTIONAL]**  It would be good to add instructions on how to run the tests in the README by adding the command to run the test so that users can easily find all commands that are needed to run your app.

Reply:
@julie-ify, thank you for your feedback. I will do it 👌